mpt/core/src/main/java/org/apache/james/mpt/protocol/ProtocolSession.java

```diff
@@ -497,6 +502,24 @@ public boolean isClient() {
         }
     }
 
+    private class ReinitElement implements ProtocolElement {
+
+        private final int sessionNumber;
+
+        public ReinitElement(int sessionNumber) {
+            this.sessionNumber = sessionNumber < 0 ? 0 : sessionNumber;
```

Comment:
I thought that you were again ternary expression ! In this case I think Math.max(0, sessionNumber) would be more readable